### ISSUE TITLE
HyperPlay 

### ISSUE THREAD

**Issue by pimentel08** [HyperPlay-Gaming/hyperplay-desktop-client#405]

**Comment by pimentel08** [comment]
@Kwilke1982 are we waiting on assets from Bushi, Cards of Ethernity, and Mini Royale Nations? Or should I just grab an image from the assets we already have from their HP game page?

**Comment by Kwilke1982** [comment]
@pimentel08 Yes, I have requested the assets from them and will share here once they are complete.

**Comment by pimentel08** [comment]
Awesome, thanks for the update!

**Comment by Kwilke1982** [comment]
![banner2.png](https://images.zenhubusercontent.com/640f8404385e8933879f4198/cfc3aceb-606c-4165-bc17-90ada9e1e0cb)![Hyperplay x Yokai Kingdom 1403x474 A.png](https://images.zenhubusercontent.com/640f8404385e8933879f4198/e13eec9b-b4ab-446a-bb5b-a33d804926b9)

**Comment by Kwilke1982** [comment]
![2023-07-10 13.59.38.jpg](https://images.zenhubusercontent.com/640f8404385e8933879f4198/d4fb2acb-838c-497f-86a1-27a97f58aa38)

**Comment by Kwilke1982** [comment]
![photo_2023-07-10 14.02.56.jpeg](https://images.zenhubusercontent.com/640f8404385e8933879f4198/8d7e2a51-2696-447a-9b39-b0ea20694542)

**Comment by Kwilke1982** [comment]
![1403x474.png](https://images.zenhubusercontent.com/640f8404385e8933879f4198/859f31bf-2036-4994-9460-07df1d16783d)

**Comment by Kwilke1982** [comment]
All banners are now uploaded and ready to go into the HP Store. @pimentel08 @BrettCleary
